lib/Config/Shortcodes.php

```diff
@@ -74,12 +83,11 @@ public function getSubscribersCount($params) {
     if (empty($segmentIds)) {
       return $this->wp->numberFormatI18n(Subscriber::filter('subscribed')->count());
     } else {
-      return $this->wp->numberFormatI18n(
-        SubscriberSegment::whereIn('segment_id', $segmentIds)
-          ->select('subscriber_id')->distinct()
-          ->filter('subscribed')
-          ->findResultSet()->count()
-      );
+      $count = 0;
+      foreach ($segmentIds as $segmentId) {
+        $count += $this->segmentSubscribersRepository->getSubscribersCount($segmentId, SubscriberEntity::STATUS_SUBSCRIBED);
```

Comment:
I'm afraid that this will not give correct results. When a subscriber is subscribed to multiple segments it will count them multiple times. On the other hand, the original code doesn't handle dynamic segments. So maybe this could be an acceptable trade-off. I guess this is something we could discuss with @NeosinneR .

Reply:
We should not count subscribers multiple times, because that could result in more subscribers than there really is. So if a subscriber is subscribed to multiple lists, he should be counted only once. I think it's fine not counting dynamic segments, as there is a high chance, that if someone is subscribed to a dynamic segment, they are also subscribed to the regular list.